openmdao/util/unittest_util.py

```diff
@@ -0,0 +1,41 @@
+
+
+def problem_derivatives_test(unittest, problem, tol = 1e-5, skip_comps = False):
+    """
```

Comment:
We have a test/util.py.  Since this is a test utility, it might be better to move it there.